main.js

```diff
@@ -3029,9 +3029,9 @@ function startInstance(id, wakeUp) {
                     ) {
                         states.setState(id + '.sigKill', {val: 0, ack: false, from: hostObjectPrefix}); // set to 0 to stop any pot. already running instances, especially broken compactModes
 
+                        const _instance = (instance && instance._id && instance.common) ? instance._id.split('.').pop() || 0 : 0;
```

Comment:
`_instance` was accessed in the else branch where it is not defined

Reply:
good catch